internal/controller/serving/reconcilers/kserve_raw_route_reconciler.go

```diff
@@ -121,15 +121,16 @@ func (r *KserveRawRouteReconciler) createDesiredResource(ctx context.Context, lo
 			}
 		}
 	}
-
-	for _, port := range targetService.Spec.Ports {
-		if !enableAuth {
-			if port.Name == "http" || port.Name == targetService.Name {
-				servicePort = port.Port
-			}
-		} else {
+	if enableAuth {
```

Comment:
same comment than here: https://github.com/opendatahub-io/kserve/pull/462/files#r1915346061

Reply:
not sure how that would help here, maybe I'm missing something? 
if auth is enabled we want to look for the `https` port and if auth is disabled we want to look for the `http`/ `<service_name>` port